contributors/devel/godep.md

```diff
@@ -15,6 +15,20 @@ the tools.
 
 This doc will focus on predictability and reproducibility.
 
+## Justifications for an update
+
+Before you update a dependency, take a moment to consider why it should be 
+updated. Valid reasons include:
+ 1. We need new functionality that is in a later version.
+ 2. New APIs in the dependency are better than the old ones.
```

Comment:
I don't think "better APIs" is meaningful on its own.  Aside from "better" being subjective, if the improvement isn't needed by something, why incur risk?

Reply:
> I don't think "better APIs" is meaningful on its own. Aside from "better" being subjective, if the improvement isn't needed by something, why incur risk?

We make judgements about "better" in nearly every pull we review.

An API that better matches expectation or is easier to reason about improves the quality of our code that consumes it.  I think that in service to improving our code quality, bumping a dependency is justified.

Reply:
I'll take "New or improved APIs in the dependency significantly improve Kubernetes code" but not simply "New or improved APIs in the dependency".  Fair?

Reply:
> I'll take "New or improved APIs in the dependency significantly improve Kubernetes code" but not simply "New or improved APIs in the dependency". Fair?

Sure.  That's more crisp.